config/autoload/global.php

```diff
@@ -57,12 +57,21 @@
         'max_photos_page' => 20,
         'dir_mode' => 0777, // rwx by default
```

Comment:
Kan misschien wel lager ja, maar het is een config file voor een reden. De hele config moet aan het einde nog even goed worden ingesteld.